epoxy-adapter/src/main/java/com/airbnb/epoxy/EpoxyVisibilityItem.java

```diff
@@ -154,9 +150,11 @@ private boolean checkAndUpdateInvisible(boolean detachEvent) {
    * component is smaller than half the viewport, when it is fully visible.
    */
   private boolean checkAndUpdateFocusedVisible() {
-    return focusedVisible =
-        sizeInScrollingDirection >= viewportSize / 2 || (visibleSize == sizeInScrollingDirection
-            && sizeInScrollingDirection < viewportSize / 2);
+    boolean focusedVisibleHeight = height >= viewportHeight / 2 || (visibleHeight == height
+        && height < viewportHeight / 2);
```

Comment:
won't `height < viewportHeight / 2` always be true here, since the `height >= viewportHeight / 2` check will have failed?

Reply:
Thanks for reviewing that part. After doing more research in that I finally re-implemented this focused/unfocused check using the area since it use not width and height.